### PR TITLE
Added I18n support and Spanish translation

### PR DESCRIPTION
**Pull request by macagua**

### CODE REVIEW

File: src/StatisticBlock/schema.js [hunk 1]
```diff
@@ -1,98 +1,206 @@
...
+  },
+  Styles: {
+    id: 'Styles',
+    defaultMessage: 'Styles',
+  },
+});
 
 const statisticSchema = {
-  title: 'Statistic item',
+  title: intl.formatMessage(messages.StatisticItem),
```
Comment (avoinea) [inline src/StatisticBlock/schema.js:113]: @macagua intl is not defined

**Review by avoinea — CHANGES_REQUESTED** [review]

File: src/StatisticBlock/schema.js (hunk 1 above)
Comment (macagua) [reply src/StatisticBlock/schema.js:113]: @avoinea I am sorry for the delay, please can you check again this "pull request" and if all ok merge it? thanks

Comment (macagua) [reply src/StatisticBlock/schema.js:113]: @avoinea Done with commit [269b52f](https://github.com/eea/volto-statistic-block/pull/9/commits/269b52f084104604f0087ec84bcc8c928021f487)

**Comment by avoinea** [comment]
@macagua Thank you! Merged in #24

**Pull request closed without merging**